src/pages/public_registration/PublicRegistrationContent.tsx

```diff
@@ -62,6 +65,7 @@ export const PublicRegistrationContent = ({ registration }: PublicRegistrationCo
       <Helmet>
         <title>{mainTitle}</title>
       </Helmet>
+      <DeletedPublicationScreenReaderInformation registration={registration} />
```

Comment:
Noe litt guffent med 2 ulike komponenter som gjør det samme bare med/uten fokus på skjermleser. En dag er de ute av synk og kan gjøre mer skade enn nytte (om teksten endrer seg, men vi glemmer å endre den ene feks). Med noe sånt kan vi være sikre på at de alltid har samme info:
```suggestion
      <Box sx={visuallyHidden}>
        <DeletedPublicationInformation registration={registration} />
      </Box>
```
Men da må det også endres litt på `aria-hidden` som settes i `DeletedPublicationInformation`. Kanskje er det en enda bedre løsning å sende en prop som sier om den er for screen reader eller ikke, og sette `aria-hidden` og `sx` basert på det.

Har ingen bestemt mening om hva som er best/riktigst, så du kan tenke litt på det, og gjøre det du selv syns gir mest mening 👍